sklearn/model_selection/_validation.py

```diff
@@ -499,7 +500,7 @@ def _index_param_value(X, v, indices):
 
 def permutation_test_score(estimator, X, y, groups=None, cv=None,
                            n_permutations=100, n_jobs=1, random_state=0,
-                           verbose=0, scoring=None):
+                           verbose=0, scoring=None, pre_dispatch='2*n_jobs'):
```

Comment:
when do we have this parameter? Always when we have n_jobs? are we consistent with that? Do we want to be?